mobile/src/main/res/layout/activity_collect_help.xml

```diff
@@ -22,5 +22,90 @@
 
     </com.google.android.material.appbar.AppBarLayout>
 
-    <include layout="@layout/content_collect_help" />
+    <FrameLayout
```

Comment:
why removing the include ? we can give an ìd`  to that layout and access it directly.